sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -144,5 +145,33 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>enable-coverage-all</id>
+      <activation>
+        <property>
+          <name>env.TEST_PACKAGES_ENABLED</name>
```

Comment:
what if there is no env.TEST_PACKAGES_ENABLED variablr at all? then we will fall back to ```<jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>```, is this what we want?

Reply:
the default is all right? It is not possible to not have a env.TEST_PACKAGES_ENABLED variable at all. 

Even if that is possible, it means that none of the tests will run in which I think it is okay to skip coverage check.

Reply:
* For live test pipeline the default is all and there is no problem.
* What about CI pipeline and release pipeline? Do we run live tests in release pipeline?

Reply:
You're right, I noticed that ci pipeline was affected by this and fixed accordingly